kebechet/managers/config_initializer/config_initializer.py

```diff
@@ -73,7 +73,9 @@ def run(self) -> typing.Optional[dict]:  # type: ignore
             with open(".thoth.yaml", "w+") as f:
                 f.write(thoth_config)
             repo.index.add([".thoth.yaml"])
-            repo.index.commit("Initialize .thoth.yaml with basic configuration")
+            repo.index.commit(
+                "Initialize .thoth.yaml with basic configuration", signoff=True
```

Comment:
The [commit method](https://gitpython.readthedocs.io/en/3.1.29/reference.html#git.index.base.IndexFile.commit) doesn't seem to understand a `signoff` argument:

```python
>>> idx.add('deleteme')
[(100644, e69de29bb2d1d6434b8b29ae775ad8c2e48c5391, 0, deleteme)]
>>> idx.commit("testing gitpython", signoff=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: commit() got an unexpected keyword argument 'signoff'
```